packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -1012,6 +1044,53 @@ contract Deploy is Deployer {
         ChainAssertions.checkSystemConfig({ _contracts: _proxies(), _cfg: cfg, _isProxy: true });
     }
 
+    /// @notice Initialize the SystemConfigInterop
+    function initializeSystemConfigInterop() public broadcast {
+        console.log("Upgrading and initializing SystemConfigInterop proxy");
```

Comment:
If you every catch yourself copy/pasting huge blocks of code like this it should be a red flag to think about the problem from a diff pov